github/info.json

```diff
@@ -2826,6 +2826,16 @@
           "tooltip": "Specify the name of the repository to be cloned on GitHub",
           "description": "Specify the name of the repository to be cloned on GitHub"
         },
+        {
+          "title": "Branch Name",
+          "name": "branch",
+          "visible": true,
+          "required": true,
```

Comment:
cannot be required...else previously configured actions in CICD SP will fail on connector upgrade.
make it optional and default to 'main'

Reply:
done